dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/expressiondimensionitem/ExpressionDimensionItemService.java

```diff
@@ -25,44 +25,45 @@
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package org.hisp.dhis.expressiondimensionitem;
+package org.hisp.dhis.dxf2.expressiondimensionitem;
 
 import java.util.ArrayList;
 import java.util.List;
 import java.util.regex.Pattern;
 
+import lombok.AllArgsConstructor;
+
 import org.hisp.dhis.category.CategoryOptionCombo;
 import org.hisp.dhis.common.BaseDimensionalItemObject;
 import org.hisp.dhis.common.DataDimensionItem;
+import org.hisp.dhis.common.IdentifiableObject;
 import org.hisp.dhis.common.IdentifiableObjectManager;
 import org.hisp.dhis.dataelement.DataElement;
 import org.hisp.dhis.dataelement.DataElementOperand;
+import org.springframework.stereotype.Service;
 
 /**
  * Parsing the expression of ExpressionDimensionItem, provides collection of
  * BaseDimensionalItemObjects.
  */
-public class ExpressionDimensionItemHelper
+@Service
+@AllArgsConstructor
```

Comment:
We avoid using `AllArgsConstructor`, and try to use `RequiredArgsConstructor` if possible.

Reply:
done